src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -87,9 +87,9 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
       '#attributes' => ['autocomplete' => 'off'],
     ];
     $form['username'] = [
-      '#type' => 'email',
+      '#type' => 'textfield',
       '#title' => $this->t('Username'),
-      '#description' => $this->t("Organization user's email address that is used for authenticating with the endpoint."),
+      '#description' => $this->t("Organization username that is used for authenticating with the endpoint."),
```

Comment:
```suggestion
      '#description' => $this->t("Apigee user's email address or identity provider username that is used for authenticating with the endpoint."),
```
Don't think we should change to username because you don't user the username if it is an Apigee User:
<img width="644" alt="Screen Shot 2019-09-25 at 9 07 37 AM" src="https://user-images.githubusercontent.com/613093/65619087-5f533600-df74-11e9-80a3-dfe4b7cf058d.png">